### DISCUSSION TITLE
SOLVED:how to force text mode in interface permanently?and.. how to create usb key for grubfm (bios and uefi)

### DISCUSSION THREAD

**Discussion by lucdeluc** [a1ive/grub2-filemanager#333] [Q&A]
is there a way to force interface using text mode permanently? maybe modding some parameters?
How to create an usb key for grubfm (bios and uefi booting capable)?
in this moment im using grubfm as an addition to ventoy but want to try this as alone boot metod
thankyou

**Comment by a1ive** [comment]
> How to create an usb key for grubfm (bios and uefi booting capable)?

See https://github.com/a1ive/grub2-filemanager#usb-installers

> is there a way to force interface using text mode permanently?

You can create the file /boot/grubfm/config on the USB drive and write the following command in it:
```
set lang=en_US
terminal_output console
```

**Reply by lucdeluc** [reply]
well, i use your grubfm_multiarch 740.iso inside ventoy , how can i force interface using text mode permanently, inside iso i cannot find a directory structure where i can put config file and then rewrite iso, some suggestions?
thankyou

**Reply by a1ive** [reply]
> create the file /boot/grubfm/config on the USB drive.

**Comment by steve6375** [comment]
E2B uses grubfm - see 
https://www.easy2boot.com/uefi-mbr-a1ive-grub2-file-manager/
also details in eBook #4

**Reply by lucdeluc** [reply]
really too complicated for me E2B, i will stay with Ventoy plus your grubfm_multiarch 740.iso (using f4 to switch to txt interface every time if no way to modify your iso)
thankyou very much for your help

**Comment by a1ive** [comment]
1. Create folder `boot` under any partition
2. Create folder `grubfm` under the `boot` folder.
3. Create an empty file named `config` under the `grubfm` folder.
4. Write the following command to `config`:
```
set lang=en_US
terminal_output console
```
Is this hard?

**Reply by lucdeluc** [reply]
you are right !!! nothing hard, i have followed your suggestion and put this boot folder into ventoy disk, the same where i put grubfm_multiarch 740.iso, and voila' all is done  txt interfarce works by default. 
that was confusing me was: where exactly put this boot folder, but now thanks to your patience the magic is done.
thankyou thankyou really for all
